packages/xrpc-server/src/server.ts

```diff
@@ -177,7 +178,6 @@ export class Server {
       middleware.push(this.middleware.json)
       middleware.push(this.middleware.text)
     }
-    this.setupRouteRateLimits(nsid, config)
```

Comment:
This line was particularly error prone:
- `setupRouteRateLimits` must be called *before* `createHandler` otherwise rate limits would silently be ignored
- `setupRouteRateLimits` sets a value in `this.routeRateLimiters[nsid]`. That value is read from `createHandler` and never used again.